README.md

```diff
@@ -395,6 +395,7 @@ section of the
 This library has no packaging or build processes. Usage of the library is
 explained in the
 [User & Programmers Manual](doc/usermanual.md).
+The following features are listed as [deprecated](https://github.com/telefonicaid/iotagent-node-lib/tree/master/doc/deprecated.md)
```

Comment:
Please add a blank line just before this in order to have a line break in the document.

Reply:
In addition, better to use a relative link for documentation, i.e.

```
[deprecated](doc/deprecated.md)
```

Reply:
Done the required changes.